<launch>

  
    <node pkg="joy" name="joy_node" type="joy_node"/>
    <node pkg="urg_node" name="urg_node" type="urg_node"/>
    <node pkg="sample_labeler" name="sample_labeler" type="picture_labeler.py" output="screen"/>
    <node pkg="cv_camera" name="cv_camera_node" type="cv_camera_node"/>
    <node pkg="image_view" name="image_view" type="image_view" output="screen">
    <!-- <remap from="image" to="/cv_camera_node/image_raw" /> -->
    <remap from="image" to="image_topic" />
    </node>
    
  

</launch>
